Flixter/TrailerViewController.m

```diff
@@ -0,0 +1,87 @@
+//
+//  TrailerViewController.m
+//  Flixter
+//
+//  Created by Angelina Zhang on 6/17/22.
+//
+
+#import "TrailerViewController.h"
+#import "UIImageView+AFNetworking.h"
+
+@interface TrailerViewController ()
+
+@property (nonatomic, strong) NSArray *movieTrailerData;
+
+@end
+
+@implementation TrailerViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    [self trailerNetworkRequest];
+}
+
+- (void)trailerNetworkRequest {
```

Comment:
IMO the call site could be improved by changing this

okay: `[self trailerNetworkRequest];`
good: `[self fetchTrailer];`
better: 
```
self.trailerRequest = [self createFetchTrailerRequest]; // the benefit is that you can also cancel the request if it's stored
[self.trailerRequest resume];
```
best:
```
// offload the work to a different object with a single responsibility
// this will help when you have more complex behavior like custom caching, whatever.
self.movieTrailer = [self.movieNetworker fetchTrailer]; 
// then can easily control the request
[self.movieNetwork cancelTrailerRequest];
```